pyproject.toml

```diff
@@ -24,27 +24,41 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.8.1"
 pydantic = "^1.10.10"
-requests = "^2.31.0"
-tiktoken = "0.4.0"
-beautifulsoup4 = "^4.12.2"
 openai = "^0.27.8"
 langchain = ">=0.0.271,<0.1.0"
-pandas = ">=1.0.5"
 pygments = "^2.15.1"
-google-api-python-client = "^2.90.0"
-chispa = "^0.9.2"
-grpcio = ">=1.56.0"
+
+[tool.poetry.group.plot.dependencies]
+pandas = ">=1.0.5"
 plotly = "^5.15.0"
-pyarrow = ">=4.0.0"
-grpcio-status = ">=1.56.0"
+
+[tool.poetry.group.vector-search.dependencies]
 faiss-cpu = "^1.7.4"
 sentence-transformers = "^2.2.2"
-torch = ">=2.0.0, !=2.0.1"
+torch = ">=2.0.0, !=2.0.1, !=2.1.0"
```

Comment:
Why it can't be 2.1.0? let's add a comment

Reply:
actually 2.1.0 is ok, but added a comment about the other version restrictions